### ISSUE TITLE
a little confusion about train and evaluate

### ISSUE THREAD

**Issue by lannester666** [huiminren/MTrajRec#3]
In the train process and evaluate process,epoch_train_id_loss is calculated.But in train id_loss calculate is epoch_train_id_loss = loss_train_ids.item(),while in evaluate is epoch_train_id_loss += loss_train_ids.item().How is this difference produced?

**Comment by huiminren** [comment]
> In the train process and evaluate process,epoch_train_id_loss is calculated.But in train id_loss calculate is epoch_train_id_loss = loss_train_ids.item(),while in evaluate is epoch_train_id_loss += loss_train_ids.item().How is this difference produced?

Thanks for your comments. It was a bug and the id loss in the training process should be '+=' .
